sdk/src/main/java/co/omise/android/models/ModelTypeResolver.kt

```diff
@@ -29,7 +29,7 @@ class ModelTypeResolver : TypeIdResolverBase() {
             (types as HashMap<String, Class<*>>)["document"] = Document::class.java
             (types as HashMap<String, Class<*>>)["authentication"] = Authentication::class.java
         }
-        return Collections.unmodifiableMap(types)
+        return Collections.unmodifiableMap(types!! as MutableMap<out String, out Class<*>>)
```

Comment:
@AnasNaouchi can we adjust to avoid this "force unwrap" of `types`? We have a `null` check above.

Reply:
@AnasNaouchi does this work?

```
    fun getKnownTypes(): Map<String, Class<*>> {
        val o = types as MutableMap<String, Class<*>>

        if (o == null) {
            o["card"] = Card::class.java
            o["token"] = Token::class.java
            o["capability"] = Capability::class.java
            o["payment_method"] = PaymentMethod::class.java
            o["source"] = Source::class.java
            o["error"] = APIError::class.java
            o["barcode"] = Barcode::class.java
            o["document"] = Document::class.java
            o["authentication"] = Authentication::class.java
        }
        return Collections.unmodifiableMap(o)
    }
```

Reply:
`Condition 'o == null' is always 'false'` so its not reachable